src/style.css

```diff
@@ -1,3 +1,66 @@
 body {
   margin: 0;
 }
+
+header {
+  font-family: Roboto, sans-serif;
+}
+
+nav {
```

Comment:
Consider using a more specific selector like '.nav' to avoid applying styles to other 'nav' elements that might be added to the page in the future.